com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/ComputeNextAppsStep.java

```diff
@@ -0,0 +1,55 @@
+package com.sap.cloud.lm.sl.cf.process.steps;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudApplicationExtended;
+import com.sap.cloud.lm.sl.cf.core.security.serialization.SecureSerializationFacade;
+import com.sap.cloud.lm.sl.cf.process.message.Messages;
+
+@Component("computeNextAppsStep")
```

Comment:
maybe computeNextAppsForDeployment?

Reply:
Nice catch! I did not even consider that we could use the parallel feature for restarts during blue-green deployments. However, if you're using it for that purpose, then you also need to change the messages. Right now they include the term "deployment", which is not correct in that case.